packages/contracts/contracts/libs/v0.8.x/minter-libs/MaxInvocationsLib.sol

```diff
@@ -351,34 +335,11 @@ library MaxInvocationsLib {
         uint256 projectId,
         address coreContract
     ) internal view returns (bool) {
-        // get max invocations from core contract
```

Comment:
as discussed - this changes functionality so let's revert the change for this function 👍

Reply:
Roger that! Updated

Reply:
actually, I think I was just confused and this was actually just fine as it was 💀